presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function (issue #9524).
+* Add support for predicate pushdown for partition symbols for window operators.
+* Remove SHOW STATS ON syntax
+* Restrict WHERE clause in SHOW STATS to filters that can be pushed down
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+* Immediately return final results to client for a failed query.
+
+Web UI
+---------------
```

Comment:
The width of these `-` blocks should be the same as the title of the section, please fix other titles too.